### PR TITLE
Use OpenShift Ansible modules instead of templates

### PR DESCRIPTION
**Pull request by aidenkeating**
Currently we are creating postgres using a template. We copy the
template onto the host machine and then perform an oc create. This
does not take advantage of the Ansible modules which are avaiable.

This change replaces the template yaml file with a set of Ansible
tasks. It also replaces the jar file for Keycloak Monitoring
Prometheus to use the latest upstream version instead of a forked
version of the source code. This new version allows an environment
variable to be specified for the Keycloak metrics directory instead
of having to modify the Keycloak configuration files.

### CODE REVIEW

**Comment by aidenkeating** [comment]
@matzew @rachael-oregan Would you mind taking a look? This is an attempt to clean up the provisioning of Keycloak. There will also need to be changes made to deprovisioning but I think that would be better in a separate PR.

# Verification steps
* Bring up the MCP using the `installer/install.sh` script, with the ansible service broker pointing at my DockerHub org `aidenkeatingrht`
* Provision the Keycloak service
* Ensure keycloak, postgres and metrics have been provisioned successfully

File: roles/provision-keycloak-apb/tasks/provision-postgres.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+- name: generate username
```
Comment (aidenkeating) [inline roles/provision-keycloak-apb/tasks/provision-postgres.yml]: Remove this

File: roles/provision-keycloak-apb/tasks/provision-postgres.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  shell: head /dev/urandom | tr -dc A-Za-z0-9 | head -c 13 ; echo ''
+  register: gen_username
+
+- name: generate password
+  shell: head /dev/urandom | tr -dc A-Za-z0-9 | head -c 13 ; echo ''
+  register: gen_password
+
+- debug: msg="{{ gen_username }}"
```
Comment (aidenkeating) [inline roles/provision-keycloak-apb/tasks/provision-postgres.yml]: Remove this

**Comment by rachael-oregan** [comment]
@aidenkeating verified changes there

**Pull request merged**